nodes/ati_ft_sensor_wrench_pub.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * Title: ati_ft_sensor_wrench_pub.cpp
+ *
+ * Description: This file is part of ati_ft_sensor package. It publishes the
+ * wrench data from the ATI force torque sensor. It also publishes the wrench
+ * data to the faked_forces_controller based on a passed parameter, serving as a
+ * generic proxy for admittance control using PickNik's admittance controller.
+ * Follow the general template to replicate this for another FT sensor.
+ *
+ * Author: Emmanuel Akita
+ * Email:  efakita@utexas.edu
+ * Created: 4 Oct, 2023
+ *
+ */
+
+#include <AtiFTSensor.h>
+#include <chrono>
+
+// ROS includes
+#include "geometry_msgs/msg/wrench_stamped.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/float64_multi_array.hpp"
+
+class ATIWrenchPublisher : public rclcpp::Node {
+  // Constructor
+ public:
+  ATIWrenchPublisher() : Node("ati_wrench_publisher") {
+    initializeNode();
+
+    // Create a timer that calls the publish function every millisecond.
+    timer_ = this->create_wall_timer(
+        std::chrono::milliseconds(1),
```

Comment:
Call the function every millisecond?  Why do you need such high rate... 
That is like if you were expecting to publish at 1000 Hz